README.md

```diff
@@ -60,24 +60,30 @@ Run it and forget it!
 
 - [It's fast](http://docs.traefik.io/benchmarks)
 - No dependency hell, single binary made with go
+- [Tiny](https://microbadger.com/images/traefik) [official](https://hub.docker.com/r/_/traefik/) official docker image
 - Rest API
-- Multiple backends supported: Docker, Swarm, Kubernetes, Marathon, Mesos, Consul, Etcd, and more to come
-- Watchers for backends, can listen for changes in backends to apply a new configuration automatically
 - Hot-reloading of configuration. No need to restart the process
-- Graceful shutdown http connections
-- Circuit breakers on backends
+- Circuit breakers, retry
 - Round Robin, rebalancer load-balancers
-- Rest Metrics
-- [Tiny](https://microbadger.com/images/traefik) [official](https://hub.docker.com/r/_/traefik/) docker image included
-- SSL backends support
-- SSL frontend support (with SNI)
+- Metrics (Rest, Prometheus, Datadog, Statd)
 - Clean AngularJS Web UI
-- Websocket support
-- HTTP/2 support
-- Retry request if network error
+- Websocket, HTTP/2, GRPC ready
+- Access Logs (JSON, CLF)
 - [Let's Encrypt](https://letsencrypt.org) support (Automatic HTTPS with renewal)
 - High Availability with cluster mode
 
+## Supported backends
+
```

Comment:
No more BoltDB?

Reply:
BoltDB is back!

Reply:
Welcome back BoltDB! 🎉 

Do you also want to add it back to the top of the README and index files, respectively?

Reply:
I simplified the top of the README using only the most used backends. So this is intended :)

Reply:
Works for me!